lessons/Early-Math/associative-property/langs/en-us/pages/true-or-false.adoc

```diff
@@ -0,0 +1,22 @@
+= True or False? (Associative Property)
+
+++++
+<style>
+div.circleevalsexp { width: auto; }
+td > .content > .paragraph > * { vertical-align: middle; }
+</style>
+++++
+
+Is the equation represented by the two Circles true or false? Explain why, and then describe what you would change to make the _false_ equations _true_.
+
+[.FillVerticalSpace,cols="^.^1a,^.^8a,^.^5a", stripes="none", options="header"]
+|===
+|	 | Circles.																	   |
+True or False? Explain.
```

Comment:
I wonder if you could make the table header incorporate the full set of directions?
True? Explain.
False? What you need to change to make it True?